### PR TITLE
DOCSP-39161 .NET updateBaseURL

### PR DESCRIPTION
**Pull request by lindseymoore**
## Pull Request Info

Jira ticket: https://jira.mongodb.org/browse/DOCSP-39161

- [Connect to App Services Backend](https://preview-mongodblindseymoore.gatsbyjs.io/realm/DOCSP-39161/sdk/dotnet/app-services/connect-to-app-services-backend/)

- Note: Commented out failing tests. Fixing them will be a part of this ticket: https://jira.mongodb.org/browse/DOCSP-39638.

### Reminder Checklist

Before merging your PR, make sure to check a few things.

- [ ] Did you tag pages appropriately?
  - genre
  - meta.keywords
  - meta.description
- [x] Describe your PR's changes in the Release Notes section
- [ ] Create a Jira ticket for related docs-app-services work, if any

### Release Notes

- .NET SDK
  - Application Services/Connect to an App Services App: Add a section on updating the base URL during runtime. 

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by dacharyc — APPROVED** [review]
LGTM - nice work on this!

File: source/sdk/dotnet/app-services/connect-to-app-services-backend.txt [hunk 1]
```diff
@@ -48,6 +48,62 @@ You can create multiple App client instances to connect to multiple
...
+the ``baseURL`` during runtime, call the 
+:dotnet-sdk:`app.UpdateBaseUriAsync() <reference/Realms.Sync.App.html#Realms_Sync_App_UpdateBaseUriAsync_System_Uri_>` 
+method:
+
+.. literalinclude:: /examples/generated/dotnet/BaseURLChange.snippet.update-base-url.cs
+   :language: csharp
+
+This API is experimental. As seen above, you must use ``#pragma warning disable Rlm001`` 
+and ``#pragma warning restore Rlm001`` to suppress the experimental errors, 
+where ``Rlm001`` is the experimental attributes's  ``diagnosticId``. You must
```
Comment (papafe) [inline source/sdk/dotnet/app-services/connect-to-app-services-backend.txt:87]: It's up to you, but I don't think you need to specify that is the diagnosticId

File: source/sdk/dotnet/app-services/connect-to-app-services-backend.txt [hunk 2]
```diff
@@ -48,6 +48,62 @@ You can create multiple App client instances to connect to multiple
...
+method:
+
+.. literalinclude:: /examples/generated/dotnet/BaseURLChange.snippet.update-base-url.cs
+   :language: csharp
+
+This API is experimental. As seen above, you must use ``#pragma warning disable Rlm001`` 
+and ``#pragma warning restore Rlm001`` to suppress the experimental errors, 
+where ``Rlm001`` is the experimental attributes's  ``diagnosticId``. You must 
+also import the following namespace at the top of your file, which contains 
+the ``Experimental`` attribute:
```
Comment (papafe) [inline source/sdk/dotnet/app-services/connect-to-app-services-backend.txt:89]: You don't need to add this namespace

File: source/sdk/dotnet/app-services/connect-to-app-services-backend.txt (hunk 1 above)
Comment (lindseymoore) [reply source/sdk/dotnet/app-services/connect-to-app-services-backend.txt:87]: Good to leave in for context.

**Comment by lindseymoore** [comment]
@papafe Thank you for the review! Addressed the last comment in this PR here: https://github.com/mongodb/docs-realm/pull/3267.

**Pull request merged**